### PR TITLE
test(ivy): fix broken tests

### PR DESCRIPTION
**Pull request by mhevery**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/core/test/render3/perf/duplicate_map_based_style_and_class_bindings/index.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
- *
- * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://angular.io/license
- */
+* @license
+* Copyright Google Inc. All Rights Reserved.
+*
+* Use of this source code is governed by an MIT-style license that can be
+* found in the LICENSE file at https://angular.io/license
+*/
```
Comment (gkalpak) [inline packages/core/test/render3/perf/duplicate_map_based_style_and_class_bindings/index.ts]: :disappointed:

**Review by rbuzzibarrios — APPROVED** [review]

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

**Review by kara — APPROVED** [review]
LGTM, but we should improve the commit message. "Fix broken tests" doesn't really explain what happened or how these changes address it.

File: packages/core/test/render3/perf/duplicate_map_based_style_and_class_bindings/index.ts (hunk 1 above)
Comment (mhevery) [reply packages/core/test/render3/perf/duplicate_map_based_style_and_class_bindings/index.ts]: Opps! fixed

**Pull request closed without merging**